netket/experimental/operator/_fermion_operator_2nd.py

```diff
@@ -657,12 +740,21 @@ def _pack_internals(operators: OperatorDict, dtype: DType):
     diag_idxs = np.array(diag_idxs, dtype=np.intp)
     off_diag_idxs = np.array(off_diag_idxs, dtype=np.intp)
     term_split_idxs = np.array(term_split_idxs, dtype=np.intp)
+    constant = sum(constants)
 
-    return orb_idxs, daggers, weights, diag_idxs, off_diag_idxs, term_split_idxs
+    return (
+        orb_idxs,
+        daggers,
+        weights,
+        diag_idxs,
+        off_diag_idxs,
+        term_split_idxs,
+        constant,
+    )
 
 
 @numba.jit(nopython=True)
-def _isclose(a, b, cutoff=1e-6):  # pragma: no cover
+def _isclose(a, b, cutoff=1e-8):  # pragma: no cover
```

Comment:
why sometime we use this one and some time the other one from numpy?

Reply:
Because numba has very limited functionality and doesn't implement isclose:
https://github.com/numba/numba/issues/5977